shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/statement/dml/InsertStatementContext.java

```diff
@@ -98,6 +98,9 @@ public InsertStatementContext(final Map<String, ShardingSphereDatabase> database
     
     private ShardingSphereSchema getSchema(final Map<String, ShardingSphereDatabase> databases, final String defaultDatabaseName) {
         String databaseName = tablesContext.getDatabaseName().orElse(defaultDatabaseName);
+        if (null == databaseName) {
```

Comment:
@zjcnb In what scenario will databaseName be empty?

Reply:
Auto send the `select verison()` command after connect Proxy, the`databaseName` are empty At this time.